docker/docker-entrypoint.sh

```diff
@@ -6,11 +6,8 @@ psql -h db -U postgres -c "CREATE ROLE onadata WITH SUPERUSER LOGIN PASSWORD 'on
 psql -h db -U postgres -c "CREATE DATABASE onadata OWNER onadata;"
 psql -h db -U postgres onadata -c "CREATE EXTENSION postgis; CREATE EXTENSION postgis_topology;"
 
-virtualenv /srv/onadata/.virtualenv
-. /srv/onadata/.virtualenv/bin/activate
-
+source /srv/.virtualenv/bin/activate
 cd /srv/onadata
-pip install -r requirements/base.pip
```

Comment:
I think the pip install still needs to be present in the `docker-entrypoint.sh`, so as no to have to rebuild the image every time you make changes to the requirements file. I use the docker compose file when developing and change is something that is constantly happening.